src/lib/stitching/gravity/stitching.ts

```diff
@@ -525,6 +529,29 @@ export const gravityStitchingEnvironment = (
           },
         },
       },
+      Show: {
+        viewingRoomsConnection: {
+          fragment: gql`
+            ... on Show {
+              viewingRoomIDs
+            }
+          `,
+          resolve: ({ viewingRoomIDs: ids }, _args, context, info) => {
+            if (ids.length === 0) return null
```

Comment:
`return []`

Reply:
Eh, so this is a weird one. First of all, `[]` isn't a valid connection, so that won't work.

We can return `null`, or return an object which is a valid empty connection. My reasoning for the somewhat short-cut version of `null`, is that while this is a connection, when stitched in this fashion, the use case is clearly a list (we're not going to be paginating thru linked viewing rooms stitched under a show, etc.).

Reply:
Ohhhh, yeah — makes sense 👍